examples/socket/README.md

```diff
@@ -0,0 +1,131 @@
+# Hello World with Unix domain socket
+
+This tutorial will demonstrate how to instrument your application with Dapr, and run it locally on your machine.
+You will deploying advanced `order` applications with [Unix domain socket](https://en.wikipedia.org/wiki/Unix_domain_socket) based on [Hello World](https://github.com/dapr/go-sdk/tree/main/examples/hello-world).
+
+There is a great performance imporvement With Unix domain socket, please notice that it does not support on Windows.
+
+## Prerequisites
+This quickstart requires you to have the following installed on your machine:
+- [Docker](https://docs.docker.com/)
+- [Go](https://golang.org/)
+
+## Step 1 - Setup Dapr
+
+Follow [instructions](https://docs.dapr.io/getting-started/install-dapr/) to download and install the Dapr CLI and initialize Dapr.
+
+## Step 2 - Understand the code
+
+The [order.go](./order.go) is a simple command line application, that implements four commands:
+* `put` sends an order with configurable order ID.
+* `get` return the current order number.
+* `del` deletes the order.
+* `seq` streams a sequence of orders with incrementing order IDs.
+
+First, the app instantiates Dapr client:
+
+```go
+    client, err := dapr.NewClientWithSocket(socket)
```

Comment:
Why duplicating the whole example just for socket connection?

Reply:
Changes are here:
```root:[examples]$ diff hello-world/order.go socket/order.go
6d5
<       "os"
16c15
<       daprPort       = "3500"
---
>       socket         = "/tmp/dapr-order-app-grpc.socket"
19,26d17
< var port string
<
< func init() {
<       if port = os.Getenv("DAPR_GRPC_PORT"); len(port) == 0 {
<               port = daprPort
<       }
< }
<
36c27
<       client, err := dapr.NewClientWithPort(port)
---
>       client, err := dapr.NewClientWithSocket(socket)
```

Use patch in example is not common I think.

Reply:
/cc @yaron2